docs/source/examples/shapes_images_positions.rst

```diff
@@ -43,4 +68,146 @@ Shapes, Images and Positions
+              x=VGroup(t1,t2,t3,t4).scale_in_place(0.7)
+              x.set_opacity(0.5)
+              x.submobjects[1].set_opacity(1)
+              self.add(x)
+
+
+.. manim:: BezierSpline
+    :save_last_frame:
+    
+    class BezierSpline(Scene):
+        def construct(self):
+            
+            np.random.seed(42)
+            area = 4
+            
+            x1 = np.random.randint(-area,area)
+            y1 = np.random.randint(-area,area)
+            p1 = np.array([x1,y1,0])
+            destination_dot1 = Dot(point=p1).set_color(BLUE)
+            
+            x2 = np.random.randint(-area,area)
+            y2 = np.random.randint(-area,area)
+            p2 = np.array([x2,y2,0])
+            destination_dot2 = Dot(p2).set_color(RED)
+            
+            
+            deltaP = p1-p2
+            deltaPNormalized = deltaP/get_norm(deltaP)
+            
+            theta = np.radians(90)
+            r = np.array(( (np.cos(theta), -np.sin(theta),      0     ),
+                           (np.sin(theta),  np.cos(theta),      0     ),
+                           (      0      ,        0      ,      0     ) ))
+            senk = r.dot(deltaPNormalized)
+            offset = 0.1
+            offset_along = 0.5
+            offset_connect = 0.25
+            
+            dest_line1_point1 = p1 + senk*offset - deltaPNormalized*offset_along
+            dest_line1_point2 = p2 + senk*offset + deltaPNormalized*offset_along
+            dest_line2_point1 = p1 - senk*offset - deltaPNormalized*offset_along
+            dest_line2_point2 = p2 - senk*offset + deltaPNormalized*offset_along
+            s1 = p1 - offset_connect*deltaPNormalized
+            s2 = p2 + offset_connect*deltaPNormalized
+            dest_line1 = Line(dest_line1_point1, dest_line1_point2)
+            dest_line2 = Line(dest_line2_point1, dest_line2_point2)
+            
+            Lp1s1 = Line(p1, s1)
+            
+            Lp1s1.add_cubic_bezier_curve(s1,
+                s1-deltaPNormalized*0.1,
+                dest_line2_point1+deltaPNormalized*0.1,
+                dest_line2_point1-deltaPNormalized*0.01 )
+            Lp1s1.add_cubic_bezier_curve(s1,
+                s1-deltaPNormalized*0.1,
+                dest_line1_point1+deltaPNormalized*0.1,
+                dest_line1_point1 )
+            
+            Lp2s2 = Line(p2, s2)
+            
+            Lp2s2.add_cubic_bezier_curve(s2,
+                s2+deltaPNormalized*0.1,
+                dest_line2_point2-deltaPNormalized*0.1,
+                dest_line2_point2 )
+            Lp2s2.add_cubic_bezier_curve(s2,
+                s2+deltaPNormalized*0.1,
+                dest_line1_point2-deltaPNormalized*0.1,
+                dest_line1_point2 )
+          
+            
+            mobjects= VGroup(Lp1s1, Lp2s2,dest_line1,dest_line2,destination_dot1,destination_dot2)
+            
+            mobjects.scale(2)
+            self.add(mobjects)
+
+Note: Here can come the UnitDot Example.
```

Comment:
These kind of notes should remain on issues or PRs on Github, they are rather confusing for users.

```suggestion
```